pkg/cinder/cronjob.go

```diff
@@ -76,6 +77,12 @@ func CronJob(
 		},
 	}
 
+	// add CA cert if defined
```

Comment:
@Akrog do we need the CA cert in the cronjob pod that basically does a DB purge?

Reply:
DB connection gets changed to TLS in follow up.

Reply:
@ASBishop I believe @stuggi is right, since cinder-manage command does a MySQL DB connection, so once that connections uses TLS the cronjob will need the CA bundle.